README.md

```diff
@@ -154,6 +155,7 @@ All the plugins are located externally with the latest update and is possible to
 [sunny-day-theme]: https://github.com/dwwmmn/micro-sunny-day
 [testaustime]: https://github.com/testaustime/testaustime-micro
 [transform]: https://github.com/SuSonicTH/micro-transform
+[tojour]: https://github.com/protostork/micro-tojour
```

Comment:
This should go before [transform] (`to` goes before `tr`)

Reply:
Good spot, thank you! Have reshuffled.